Umbraco-Cloud/Databases/SQL-Management-Studio/index.md

```diff
@@ -0,0 +1,25 @@
+## Connecting to SQL Azure on Umbraco Cloud Using SQL Management Studio
```

Comment:
I think this article could use a main heading, like "Connect to your database" ..
And then a short introduction as to what's in this article?